### PR TITLE
Added Image Component

### PR DESCRIPTION
**Pull request by SukanyaSen530**
Added Image Component
   a. Responsive Image
   b. Round Images

Images1

### CODE REVIEW

File: styles/core/utilities.css [hunk 1]
```diff
@@ -12,6 +12,11 @@
   justify-content: center;
 }
 
+.flex-wrap {
+  flex-wrap: wrap;
+  row-gap: 10px;
+}
+
 .max-width-height {
   width: 100vh;
```
Comment (MaruthiPothuganti) [inline styles/core/utilities.css]: Typo: Maybe width should be 100vw as it is the max-width

File: docs.html [hunk 2]
```diff
@@ -309,7 +309,7 @@ <h5>Letter Avatars</h5>
       <p>Letter avatars can be created with class <span class="code-highlight"> .avatar-letter</span>. Add the color
         classes to apply color to the avatars. This can be used with maximum of 2 letters. </p>
 
-      <div class="component-display flex-row flex-center">
+      <div class="component-display flex-row flex-center flex-wrap">
```
Comment (MaruthiPothuganti) [inline docs.html:312]: This is Awesome. You have made use of the utility classes perfectly.

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: Max width value

File: styles/core/utilities.css (hunk 1 above)
Comment (SukanyaSen530) [reply styles/core/utilities.css]: Thanks for pointing that out

Comment (SukanyaSen530) [reply styles/core/utilities.css]: fixed - 5653b94ff86ab74afb718f960495227fa419cb64

**Review by shashikantmaurya0007 — APPROVED** [review]
Image is responsive .. approving the pr things looks clean and beautiful:)

File: styles/core/utilities.css (hunk 1 above)
Comment (MaruthiPothuganti) [reply styles/core/utilities.css]: Okay to merge

**Pull request merged**